DayCare/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.contrib.auth.models import User
+from .validators import validate_username_already_exist
+from django.core.validators import validate_email
+from django.core.validators import DecimalValidator
+from django.core.validators import MaxLengthValidator
+from django.utils import timezone
+
+
+class daycare(models.Model):
```

Comment:
Please change the class name according to python name conventions
https://realpython.com/python-pep8/#naming-styles

Reply:
at first, the name of the class and the name of the app were: `DayCare` and it was made a lot of errors in the app.
 so I changed the class name only to `daycare` and not the app name because if I will need to change the app name to daycare instead I will need to re-create the app again in order to bootstrap the owl settings for the app that Django creates for us.

Reply:
GitHub didn't changed the file DayCare to daycare, @ormergi can you changed it in order pytest will pass the all tests?